miriway_commands.cpp

```diff
@@ -52,7 +52,7 @@ auto miriway::ShellCommands::keyboard_shortcuts(MirKeyboardEvent const* kev) ->
         return false;
 
     auto const mods = mir_keyboard_event_modifiers(kev);
-    auto const key_code = mir_keyboard_event_key_code(kev);
+    auto const key_code = mir_keyboard_event_keysym(kev);
```

Comment:
Maybe the variable should be renamed too?
```suggestion
    auto const keysym = mir_keyboard_event_keysym(kev);
```

Reply:
On seconds thoughts, there's a load of `xkb_keysym_t key_code` in the codebase. So "prior art"...